pkgs/development/python-modules/geopy/default.nix

```diff
@@ -1,31 +1,33 @@
 { stdenv
 , buildPythonPackage
-, fetchPypi
-, isPy27
+, fetchFromGitHub
+, isPy3k
 , mock
 , tox
 , pylint
+, geographiclib
 }:
 
 buildPythonPackage rec {
-  pname = "geopy";
-  version = "1.20.0";
-  disabled = !isPy27;
+    pname = "geopy";
```

Comment:
```suggestion
    pname = "geopy-unstable";
```

Reply:
Didn't understand the first time, thanks